.github/workflows/publish-on-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Cookie Policy release
+
+# Only for testing. Should be on: release: types: [published]
+on:
+  - push
+
+jobs:
+  build:
+    name: Build Cookie Policy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Read cookie-policy version from package.json
+        run: |
+          node -p "require('./package.json').version" > CURRENT_VERSION
+          echo "Building Cookie Policy v$(cat CURRENT_VERSION)"
+      - run: yarn install
+      - run: yarn build
+      - run: yarn test
+      - name: Show size of the build file
+        run: stat -c '%s' build/css/build.css
+      - run: cp CURRENT_VERSION build/css
```

Comment:
This also is likely Vanilla specific (we need this for publishing on assets server)

Reply:
Definitely, I left it in because it seemed interesting, but it makes the process a bit longer